scanners/boostsecurityio/baseline/module.yaml

```diff
@@ -3,10 +3,11 @@ api_version: 1.0
 
 id: boostsecurityio/scanner
```

Comment:
Sorry my other comment wasn't clear that it included this line,  this ID should be `boostsecurityio/baseline` else we're going to clash with the actual scanner